### DISCUSSION TITLE
Unsupported protocol scheme

### DISCUSSION THREAD

**Discussion by NikhilkaushikOp** [projectdiscovery/nuclei#1515] [Q&A] [answered]
Please Help Me out ---

![image](https://user-images.githubusercontent.com/93500332/150100409-b4d46560-4af5-423b-8a55-c091bc94aff0.png)

**Comment by ehsandeep** [comment, marked as answer]
You need to input urls of subdomains instead of list of subdomains, check the example input:- https://github.com/projectdiscovery/nuclei/#running-nuclei

**Reply by NikhilkaushikOp** [reply]
Thanks for that . But i want to use subdomains from subfinder tool. But the subdomains are not in http form

**Reply by ehsandeep** [reply]
`subfinder | httpx | nuclei -tags takeover` this is what you can do it automate this.

**Comment by NikhilkaushikOp** [comment]
Thank you
